### ISSUE TITLE
Python 3.9 DeprecationWarning for Python 3.10

### ISSUE THREAD

**Issue by vanjohnson879** [vilarneto/pyrtable#6]
While using `pyrtable` in 3.9 the following warning is issued.  Python 3.10 was just released and, as it states, this will not work.

```
python3.9/site-packages/pyrtable/fields/valueset.py:12: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.10 it will stop working
    class ValueSet(collections.MutableSet, collections.Iterable, Generic[T]):
```

P.S. really enjoy using this in our integration projects with Airtable!

**Comment by vilarneto** [comment]
Hello @vanjohnson879, thank you for the report. Just uploaded v0.8.2 – would you care to test it?

**Comment by vilarneto** [comment]
Fixed in 2ac50f3 (v0.8.2).

**Comment by vanjohnson879** [comment]
Yes! The warning is gone.  I'll test it in 3.10 soon.
